modules/installing-gcp-post-cluster-creation.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// * installing-gcp-user-defined-labels-and-tags.adoc
+
+:_content-type: CONCEPT
+[id="installing-gcp-post-cluster-creation_{context}"]
+= Configuration of labels and tags for GCP after the cluster creation
+
+After creating the {product-title} cluster, you can access the list of the labels and tags defined for the GCP resources by referring to the new fields, `.status.platformStatus.gcp.resourceLabels` and `.status.platformStatus.gcp.resourceTags` present in the `infrastructures.config.openshift.io/cluster` object.
+
+.Sample `infrastructure.yaml` file
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Infrastructure
+metadata:
+ name: cluster
+spec:
+ platformSpec:
+   type: GCP
+status:
+ infrastructureName: <cluster_id><1>
+ platform: GCP
+ platformStatus:
+   gcp:
+     resourceLabels:
+     - key: <label_key>
+       value: <label_value>
+     resourceTags:
+     - key: <tag_key_short_name>
+       parentID: <OrganizationID/ProjectID>
+       value: <tag_value_short_name>
+   type: GCP
+----
+<1> The `cluster_id` parameter has the value of `.status.infrastructureName` present in `infrastructures.config.openshift.io/cluster`.
```

Comment:
with reference to the above sentence.

```suggestion
<1> The `cluster_id` parameter has the value of `.status.infrastructureName` present in `infrastructures.config.openshift.io/cluster` object.
```

Reply:
Is this something that user has to do or will be auto-populated?

Reply:
It is not auto-populated.

Reply:
Then reword:
```suggestion
<1> Specify the value of `.status.infrastructureName` present in `infrastructures.config.openshift.io/cluster`.
```

Reply:
My apologies for the confusion. It is auto-populated. I was about to change the comment. So I will keep the sentence the way it is.